snapcraft/internal/states/_prime_state.py

```diff
@@ -19,15 +19,15 @@
 from snapcraft.internal.states._state import State
 
 
-def _strip_state_constructor(loader, node):
+def _prime_state_constructor(loader, node):
     parameters = loader.construct_mapping(node)
-    return StripState(**parameters)
+    return PrimeState(**parameters)
 
-yaml.add_constructor(u'!StripState', _strip_state_constructor)
+yaml.add_constructor(u'!PrimeState', _prime_state_constructor)
 
 
-class StripState(State):
-    yaml_tag = u'!StripState'
+class PrimeState(State):
+    yaml_tag = u'!PrimeState'
```

Comment:
yeah, this is what I mentioned in the standup. It would be as if it never existed, given that it is an easy step to get back on track I thought I'd let it be